pennylane/operation.py

```diff
@@ -582,6 +582,18 @@ def grad_method(self):
     transforms that merge adjacent rotations.
     """
 
+    @property
+    def as_rot_angles(self):
+        """The parameters required to implement a single-qubit gate as an
+        equivalent ``Rot`` gate, up to a global phase.
+
+        Returns:
+            [float, float, float]: A list of values :math:`[\phi, \theta, \omega]`
+            such that :math:`RZ(\omega) RY(\theta) RZ(\phi)` is equivalent to the
+            original operation.
+        """
+        raise NotImplementedError
+
```

Comment:
This is indeed covered in `test_single_qubit_fusion_not_implemented` using the `PauliRot` gate.

Reply:
:+1: